ADATOK.CPP

```diff
@@ -114,17 +114,20 @@ void encode_framecount(recorder* prec) {
 }
 
 // Check that the framecount matches with the MSB fo the flags
-/*int framecount_integrity( recorder* prec ) {
-    if( prec->betoltve < 80 )
+int framecount_integrity(recorder* prec) {
+    if (prec->betoltve < 80) {
         return 1;
+    }
     unsigned int framecount = 0;
-    for( int i = 0; i < 32; i++ ) {
+    for (int i = 0; i < 32; i++) {
         framecount *= 2;
-        if( prec->pgazhatra[40+31-i] & 128 )
+        if (prec->pgazhatra[40 + 31 - i] & 128) {
             framecount += 1;
+        }
     }
-    if( framecount == prec->betoltve )
+    if (framecount == prec->betoltve) {
```

Comment:
The diff in this commit looks very messy: 93e6b28e673ba79410af3dbb464de6cdbb57319b

Reply:
I'm not sure either but a lot of it seems to be formatting changes. I never looked at the commit past that.